app/konnect/dev-portal/developers/dev-reg-app-service.md

```diff
@@ -23,15 +23,10 @@ applicable Services.
 1. Log in to the {{site.konnect_short_name}} Dev Portal to access the Service
 Catalog.
 
-2. Click on a Service tile.
-
-   **Note:** Not all Services might have app registration enabled. Contact your {{site.konnect_short_name}} admin
-   if you need it enabled on a Service.
+2. Click on a Service tile. Not all Services may have app registration enabled. Contact your {{site.konnect_short_name}} admin if you need it enabled on a Service.
```

Comment:
```suggestion
2. Click on a Service tile. 

    Not all Services may have app registration enabled. Contact 
    your {{site.konnect_short_name}} admin if you need it enabled on a Service.
```
Separate out the instruction from the additional info where it's not a necessary part of the step.